### PR TITLE
added digitalocean svgs and sketch master file

### PR DESCRIPTION
**Pull request by Chippd**
Added digital ocean icons, icon fonts need to be added.

### CODE REVIEW

File: icons/digitalocean/digitalocean-original-wordmark.svg [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="128px" height="128px" viewBox="0 0 128 128" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 43.2 (39069) - http://www.bohemiancoding.com/sketch -->
```
Comment (badabam) [inline icons/digitalocean/digitalocean-original-wordmark.svg]: Cleanup comments and unused lines like title, desc, defs?

File: icons/digitalocean/digitalocean-original.svg [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="128px" height="128px" viewBox="0 0 128 128" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 43.2 (39069) - http://www.bohemiancoding.com/sketch -->
```
Comment (badabam) [inline icons/digitalocean/digitalocean-original.svg]: Cleanup unused lines?

File: icons/digitalocean/digitalocean-original-wordmark.svg (hunk 1 above)
Comment (amacado) [reply icons/digitalocean/digitalocean-original-wordmark.svg:3]: yeap, would be nice! :)

**Comment by Thomas-Boi** [comment]
Hello there,
Recently, we just introduced an automation pipeline to Devicon which will help us create the icons. Before we can merge your changes in, there are a few things that we need to do first:

1. Merge the current devicon/develop branch into your PR branch. This contains the build script needed for us to build your icons.
2. Read the `CONTRIBUTING.md` for things to do before merging and review our icon standards.
3. Push your changes and/or branches. We will review them once this is done and merge your PR if everything goes well.


Thank you for contributing to this repo. We are sorry that we haven't accept any PR for awhile. Now that we have a build pipeline, we can merge PRs more quickly than before. 

We hope to see your icons in the Devicon repo soon :)

Cheers,
Devicon Team

**Comment by Thomas-Boi** [comment]
Hello there,

We notice that this PR has been inactive for awhile now. It would be a waste of your time and effort if we just close this PR and throw away the SVGs in here. Thus, 1 month from now, if this PR remains inactive/stale, we will fork your repo, fix any issues, and merge your icons ourselves.

Since GitHub tracks contributions by commits, you will still be credited. We also hope that you will help us merge this PR before the deadline comes.

Cheers,
Devicon Team

**Comment by Panquesito7** [comment]
#547.

**Pull request closed without merging**